tieredhashing/tiered_hashing.go

```diff
@@ -27,7 +27,7 @@ const (
 	reasonCorrectness = "correctness"
 
 	// use rolling windows for latency and correctness calculations
-	latencyWindowSize     = 100
+	latencyWindowSize     = 50
```

Comment:
this seems unrelated to the larger correctness change? can we do this as a different PR / test them independently?